libs/application/ui-fields/src/lib/TableRepeaterFormField/TableRepeaterFormField.tsx

```diff
@@ -57,13 +58,37 @@ export const TableRepeaterFormField: FC<Props> = ({
     editButtonTooltipText = coreMessages.editFieldText,
     editField = false,
     maxRows,
+    onSubmitLoad,
+    loadErrorMessage,
   } = data
 
+  const apolloClient = useApolloClient()
+  const [loadError, setLoadError] = useState<boolean>(false)
+
   const items = Object.keys(rawItems).map((key) => ({
     id: key,
     ...rawItems[key],
   }))
 
+  const load = async () => {
+    if (onSubmitLoad) {
+      try {
+        setLoadError(false)
+        const submitResponse = await onSubmitLoad({
+          apolloClient,
+          application,
+          tableItems: values,
+        })
+
+        submitResponse.dictionaryOfItems.forEach((x) => {
+          methods.setValue(x.path, x.value)
+        })
+      } catch {
+        setLoadError(true)
+      }
+    }
```

Comment:
Early return to reduce depth
```suggestion
    if (!onSubmitLoad) {
      return
    }
    
    try {
      setLoadError(false)
      const submitResponse = await onSubmitLoad({
        apolloClient,
        application,
        tableItems: values,
      })

      submitResponse.dictionaryOfItems.forEach((x) => {
        methods.setValue(x.path, x.value)
      })
    } catch {
      setLoadError(true)
    }  
```